Source/WebCore/platform/graphics/PlatformTimeRanges.h

```diff
@@ -165,4 +166,13 @@ std::optional<PlatformTimeRanges> PlatformTimeRanges::decode(Decoder& decoder)
 
 } // namespace WebCore
 
+namespace WTF {
```

Comment:
@jernoble, I fear that I'm committing some kind of layer violation by declaring WTF stuff in a WebCore header. Can you please have a look at the new patch and approve the PR here in GitHub only if adding WTF extra declarations in PlatformTimeRanges is the right thing to do?

Reply:
The layering violation would be declaring WebCore things non a WTF header. :) 

What you're doing here is completely fine, and is a common pattern in other areas of the WebKit codebase.